Activate on workspaces with `jest.config.{mjs|cjs|ts}`

felipecrs opened the pull request:
Like https://github.com/facebook/docusaurus

felipecrs commented on package.json:
```diff
@@ -32,6 +32,7 @@
     "workspaceContains:jest.config.js",
+    "workspaceContains:jest.config.mjs",
     "workspaceContains:jest.json",
     "workspaceContains:node_modules/.bin/jest",
     "workspaceContains:node_modules/react-scripts/node_modules/.bin/jest",
```
PS: I think vscode does not watch these folders. Perhaps because there is a blacklist, or something.

Araxeus commented:
I think that for clarity this PR should be renamed to `Fix extension not activating if jest.config is .mjs`
also technically it could also be `.ts` too

felipecrs commented:
> I think that for clarity this PR should be renamed to `Fix extension not activating if jest.config is .mjs` also technically it could also be `.ts` too

Added `.ts` and `.cjs`. Also fixed the PR title, I agree.

connectdotz reviewed (approved):
looks good, thanks @felipecrs
